### ISSUE TITLE
carbon-aggregator (instance a) is already running with pid 51

### ISSUE THREAD

**Issue by tinuva** [graphite-project/docker-graphite-statsd#96]
When restarting my graphite-statsd container, it looks like it leaves the .pid files behind in the storage folder. This then cause my logs to be spammed with the following messages:
```
graphite        | carbon-aggregator (instance a) is already running with pid 51
graphite        | carbon-aggregator (instance a) is already running with pid 51
graphite        | carbon-aggregator (instance a) is already running with pid 51
graphite        | carbon-aggregator (instance a) is already running with pid 51
```

Currently using the following docker-compose to set up graphite-statsd:
```
  graphite:
    image: graphiteapp/graphite-statsd:latest
    container_name: graphite
    restart: always
    hostname: graphite
    ports:
      - 2003:2003
      - 2004:2004
      - 2023:2023
      - 2024:2024
    volumes:
      - ${USERDIR}/docker/graphite/data/graphite/conf:/opt/graphite/conf
      - ${USERDIR}/docker/graphite/data/graphite/storage:/opt/graphite/storage
      - ${USERDIR}/docker/graphite/data/graphite/log/graphite:/var/log/graphite
      - ${USERDIR}/docker/graphite/data/graphite/log/carbon:/var/log/carbon
    labels:
      - "traefik.enable=true"
      - "traefik.backend=graphite"
      - "traefik.frontend.rule=Host:graphite.${DOMAINNAME}"
      - "traefik.frontend.redirect.entryPoint=https"
      - "traefik.port=80"
      - "traefik.protocol=http"
      - "traefik.docker.network=traefik_proxy"
      - com.centurylinklabs.watchtower.enable=true
```

I can then stop the container, remove the files carbon-aggregator-a.pid and carbon-cache-a.pid from ${USERDIR}/docker/graphite/data/graphite/storage then start the container again, and the messages will gone, until next container restart. 

Is there something I can configure to prevent the above errors?

Thank you in advance.

**Comment by deniszh** [comment]
Stale pids should be removed when the container starts. I will have a look.

**Comment by deniszh** [comment]
@tinuva : should be fixed in v1.1.5-11, please test
